images/janusgraph/hack/files/submit-reindex-jobs.groovy

```diff
@@ -21,24 +21,22 @@ indexes = [
 
 //Wait for the index to become available
 indexes.each { i ->
-   index = mgmt.getGraphIndex(i)
-   propertykeys = index.getFieldKeys()
-   propertykeys.each { j ->
-   indexof = propertykeys.findIndexOf{it ==~ 'j'}
-   indexcurrentstatus = index.getIndexStatus(propertykeys[indexof])
-   if (indexcurrentstatus == SchemaStatus.ENABLED)
+  index = mgmt.getGraphIndex(i)
+  propertykeys = index.getFieldKeys()
+  propertykeys.each { j ->
+  indexof = propertykeys.findIndexOf{it ==~ 'j'}
+  indexcurrentstatus = index.getIndexStatus(propertykeys[indexof])
+  if (indexcurrentstatus == SchemaStatus.ENABLED)
    {
     System.err.println "Schema Status in ENABLED"
    }
-   else    
+   else if (indexcurrentstatus == SchemaStatus.INSTALLED)  
    { 
    mgmt.awaitGraphIndexStatus(graph, i).status(SchemaStatus.REGISTERED).call()   
```

Comment:
Le'ts put a debug message as in the previous branch - something like "Schema status is installed" so we know in the logs what's going on.